src/twisted/test/test_rebuild.py

```diff
@@ -43,55 +45,49 @@ class RebuildTests(unittest.TestCase):
     def setUp(self):
         self.libPath = self.mktemp()
         os.mkdir(self.libPath)
-        self.fakelibPath = os.path.join(self.libPath, 'twisted_rebuild_fakelib')
+        self.fakelibPath = os.path.join(self.libPath,
+                                        'twisted_rebuild_fakelib')
         os.mkdir(self.fakelibPath)
         open(os.path.join(self.fakelibPath, '__init__.py'), 'w').close()
         sys.path.insert(0, self.libPath)
 
+
     def tearDown(self):
         sys.path.remove(self.libPath)
 
-    def testFileRebuild(self):
+
+    def test_FileRebuild(self):
         from twisted.python.util import sibpath
         import shutil, time
         shutil.copyfile(sibpath(__file__, "myrebuilder1.py"),
                         os.path.join(self.fakelibPath, "myrebuilder.py"))
         from twisted_rebuild_fakelib import myrebuilder
         a = myrebuilder.A()
-        try:
-            object
-        except NameError:
+        from twisted.test import test_rebuild
+        b = myrebuilder.B()
+        class C(myrebuilder.B):
             pass
-        else:
-            from twisted.test import test_rebuild
-            b = myrebuilder.B()
-            class C(myrebuilder.B):
-                pass
-            test_rebuild.C = C
-            C()
+        test_rebuild.C = C
+        C()
         i = myrebuilder.Inherit()
         self.assertEqual(a.a(), 'a')
-        # necessary because the file has not "changed" if a second has not gone
+        # Necessary because the file has not "changed" if a second has not gone
         # by in unix.  This sucks, but it's not often that you'll be doing more
         # than one reload per second.
         time.sleep(1.1)
         shutil.copyfile(sibpath(__file__, "myrebuilder2.py"),
                         os.path.join(self.fakelibPath, "myrebuilder.py"))
         rebuild.rebuild(myrebuilder)
-        try:
-            object
-        except NameError:
-            pass
-        else:
-            b2 = myrebuilder.B()
-            self.assertEqual(b2.b(), 'c')
-            self.assertEqual(b.b(), 'c')
+        b2 = myrebuilder.B()
+        self.assertEqual(b2.b(), 'c')
+        self.assertEqual(b.b(), 'c')
         self.assertEqual(i.a(), 'd')
         self.assertEqual(a.a(), 'b')
-        # more work to be done on new-style classes
+        # More work to be done on new-style classes
         # self.assertEqual(c.b(), 'c')
```

Comment:
Understood on the PR scope - I was just wondering if these unfinished bits of code in here (that only appear to exercise code that is already exercised by other tests, and make the purpose of the test slightly confusing) could just be removed, with "tidying up" being within scope of the PR... ?

This test appears to have been written to support Python versions when the whole new/old-style issue was different (pre-Py3) and it looks to me like the testing needs addressed here actually need to be rethought as a whole, and this fragment of non-functional test does not really help in this respect.

Not that I am any sort of expert, I'm just feeling out the process and depth/scope of PR reviews here - so am looking for feedback on my comments as much as anything else.